apisix/core/ctx.lua

```diff
@@ -159,10 +160,22 @@ do
                     val, err = cookie:get(sub_str(key, 8))
                     if err then
                         log.warn("failed to fetch cookie value by key: ",
-                                 key, " error: ", err)
+                                key, " error: ", err)
```

Comment:
Please revert this change, the original indent is good.

Reply:
ok

Reply:
bad indent

Reply:
```suggestion
                                 key, " error: ", err)
```